### ISSUE TITLE
magboots sprite is goneski

### ISSUE THREAD

**Issue by exp111** [space-wizards/space-station-14#2161]
![grafik](https://user-images.githubusercontent.com/17885980/94917044-151e6200-04b0-11eb-9cc3-b89109868469.png)

```
Unable to load RSI '/Textures/Clothing/Shoes/magboots0.rsi'. Trace:
System.IO.FileNotFoundException: Path does not exist in the VFS: '/Textures/Clothing/Shoes/magboots0.rsi/meta.json'
```

**Comment by FirinMaLazors** [comment]
I'll see if I can do this!

**Comment by SweptWasTaken** [comment]
Fixed by #2156
